src/form.svelte

```diff
@@ -12,13 +12,36 @@
   let saveState = "";
   let errorMessage = "";
   let availableTagNames = []
+  let formTitle = "";
 
   async function init() {
     const tabInfo = await getCurrentTabInfo();
     url = tabInfo.url;
     titlePlaceholder = tabInfo.title;
     const availableTags = await getTags().catch(() => [])
     availableTagNames = availableTags.map(tag => tag.name)
+
+   initCurrentBookmarkData();
+  }
+
+  async function initCurrentBookmarkData() {
+    formTitle = 'Searching for existing Bookmarks';
```

Comment:
This is kind of noisy, as the element first shows this message, and then quickly switches to the new message. Would suggest to initialize the `formTitle` variable with `Add bookmark` instead, as that covers 99% of cases.